basicsettings.ml

```diff
@@ -99,5 +91,27 @@ let optimise = Settings.add_bool("optimise", false, `User)
 (* Compile & cache whole program, closures, and HTML *)
 let cache_whole_program = Settings.add_bool("cache_whole_program", false, `User)
 
```

Comment:
Many of these flags were only useful in experimental evaluation/performance profiling, it may make sense to remove them and specialize the underlying shredding code with sensible default values.